packages/vite-plugin-cloudflare/src/assets/asset-worker.ts

```diff
@@ -0,0 +1,44 @@
+// @ts-ignore
+import AssetWorker from '@cloudflare/workers-shared/dist/asset-worker.mjs';
```

Comment:
The workers from `@cloudflare/workers-shared` are not currently properly exported with types. When they are, they types in this file will be improved.